dashboard/test/models/script_test.rb

```diff
@@ -267,8 +267,10 @@ def populate_cache_and_disconnect_db
       refute s.hidden?, "#{s.name} is hidden when it should not be"
     end
 
+    # filter out factory created scripts
+    all_scripts = Script.all.select {|s| !(/^bogus-script.*/ =~ s.name)}
```

Comment:
This feels like possible spooky action at a distance.  Can we share a constant with `factories.rb`, or somehow define these side-by-side?

Reply:
I'd asked about this test in dashboard-dev, but didn't get any response.

I *think* the entire test is actually useless. It was originally created for us to catch when content writers accidentally toggled the hidden state of a script. These days, we get these all from fixtures (and apparently factory-created scripts, depending on when you create them). As such, I don't think the test gives us any value, and I should probably just remove the entire thing.

Agree/disagree?

Reply:
We're... using tests to catch changes to our fixtures?  Sure, cut it.

If we want to test which scripts are hidden, we should probably do that as late as possible in an acceptance (UI) test.